easybuild/easyconfigs/v/vcf-inheritance-matcher/vcf-inheritance-matcher-v0.1.1-Java-11-LTS.eb

```diff
@@ -0,0 +1,33 @@
+name = 'vcf-inheritance-matcher'
+version = 'v0.1.1'
+versionsuffix = '-Java-%(javaver)s-LTS'
+
+homepage = 'https://github.com/molgenis/vip-inheritance-matcher/'
+description = """Command-line application to match gene inheritance modes with sample inheritance based on genmod annotations."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
Dit werkt nog wel maar de `dummy` toolchain is deprecated en gaat er op termijn uit. De laatste versie van EB die we op Gearshift hebben kan al met het nieuwere:
`toolchain = SYSTEM`
overweg, dus het wordt tijd om de syntax te updaten...

Reply:
ok, nemen we mee voor de volgende release.